src/controllers/dynakube/oneagent/daemonset/daemonset.go

```diff
@@ -213,18 +213,12 @@ func (dsInfo *builderInfo) podSpec() corev1.PodSpec {
 		TerminationGracePeriodSeconds: address.Of(defaultTerminationGracePeriod),
 	}
 
-	if dsInfo.dynakube.NeedsOneAgentReadinessProbe() {
-		podSpec.Containers[0].ReadinessProbe = &corev1.Probe{
-			ProbeHandler: corev1.ProbeHandler{
-				Exec: &corev1.ExecAction{
-					Command: dsInfo.dynakube.Status.OneAgent.Healthcheck.Test,
-				},
-			},
-			InitialDelaySeconds: DefaultReadinessProbeInitialDelay,
-			PeriodSeconds:       int32(dsInfo.dynakube.Status.OneAgent.Healthcheck.Interval.Seconds()),
-			TimeoutSeconds:      int32(dsInfo.dynakube.Status.OneAgent.Healthcheck.Timeout.Seconds()),
-			FailureThreshold:    int32(dsInfo.dynakube.Status.OneAgent.Healthcheck.Retries),
-		}
+	if dsInfo.dynakube.NeedsOneAgentProbe() {
+		podSpec.Containers[0].ReadinessProbe = dsInfo.getProbe()
```

Comment:
```suggestion
		podSpec.Containers[0]. LivenessProbe = dsInfo.getProbe()
```
Why not combine it?

Reply:
you are right, thanks. i got confused by the condition that liveness probe should only be set if readiness is set. :slightly_smiling_face:

Reply:
fixed in [9863db2](https://github.com/Dynatrace/dynatrace-operator/pull/2115/commits/9863db2f7e301ed91d3595575008084c88a3ce09)